docs/circuit_cutting/tutorials/01_gate_cutting_to_reduce_circuit_width.ipynb

```diff
@@ -322,9 +322,17 @@
     "    jobs = {\n",
```

Comment:
This probably a little unrelated to the scope of this PR but I wanted to point this out anyway. I encountered this while running something in batch mode recently. 
1. Should `samplers` be defined inside the context manager specified by `Batch`? I noticed that when I did:

```
estimators = {edge: Estimator(backend=backend) for edge in list_of_edges}
with Batch(backend=backend): 
     jobs = {edge: estimator.run([(xpiled_circuit, experiments[edge])])
        for edge, estimator in estimators.items()}
```
my jobs did not get a batch id. 

    
   2. Secondly, do we need independent instantiations of `SamplerV2` for each subexperiment?

Reply:
> do we need independent instantiations of SamplerV2 for each subexperiment?

No, we don't.  This is legacy from when we wanted to give the user an option to run across two different hardware devices.  But this is not compatible with `Batch` mode anyway.

If we have a single sampler created outside the context manager, does this fix point 1?

Reply:
>If we have a single sampler created outside the context manager, does this fix point 1?

I would have to test this out but what I suggested above was recommended to me by a colleague (Saki).

Reply:
I got rid of the separate samplers.  I am still creating the sampler outside of the context manager because `batch.session.id` seems to get set either way.  If we find that there is an additional improvement to be made here, we can always make a follow up PR.

Reply:
Okay, that makes sense.